### DISCUSSION TITLE
MQTT password

### DISCUSSION THREAD

**Discussion by PedroKTFC** [johanmeijer/grott#500] [Q&A]
When I run the Grott server as an HA add-on, the MQTT account used is:
> mqttuser: addons
> mqttpsw: \*\*secret * *

whereas if I run it locally it's the default:
> mqttuser: grott
> mqttpsw: \*\*secret**

I can't remember ever creating a user `addons` and I certainly don't remember its password. Is there a elatively easy way of finding out the password? The former updates my HA sensors while the default (not surpringly) doesn't.
Thanks

**Comment by johanmeijer** [comment]
Hmm hopefully you can not retrieve the password, that would be very
insecure. I think you have to make a new password (or new user?). I am not
sure how to do this with the integrated HA MQTT.

Op wo 27 dec 2023 om 11:13 schreef PedroKTFC ***@***.***>:

**Comment by PedroKTFC** [comment]
The add-on is picking up the user name and pw from, I'm guessing, an environment variable? As far as I can tell, it's not picking up any encrypted version. I'll try to dig deeper to find out where it's getting it from. Thanks
